### PR TITLE
gpg-tui 0.8.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> /opt/homebrew/Cellar/gpg-tui/0.8.0/bin/completions
Failed to execute: /opt/homebrew/Cellar/gpg-tui/0.8.0/bin/completions
```

**Comment by cho-m** [comment]
Upstream renamed that binary to `gpg-tui-completions`: https://github.com/orhun/gpg-tui/commit/c8c8adc24878675e90f8677fd1673857d75d538c

File: Formula/gpg-tui.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GpgTui < Formula
   desc "Manage your GnuPG keys with ease! 🔐"
   homepage "https://github.com/orhun/gpg-tui"
-  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.7.4.tar.gz"
-  sha256 "12c61fac67f6f9b90c9f92e5ae754652b024ea3f85b99a2ac359f44356c49467"
+  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.8.0.tar.gz"
```
Comment (cho-m) [inline Formula/gpg-tui.rb:4]: Might be chance to change to more common style:
```suggestion
  url "https://github.com/orhun/gpg-tui/archive/v0.8.0.tar.gz"
```
There are only ~100 formulae that include `refs/tags` vs ~2000 that use shorter URL

Comment (chenrui333) [reply Formula/gpg-tui.rb:4]: yeah, I was thinking about that as well.

**Review by cho-m — APPROVED** [review]

**Pull request closed without merging**